content/en/monitors/guide/monitoring-sparse-metrics.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Monitoring Sparse Metrics
+kind: guide
+further_reading:
+- link: "/api/latest/monitors/#edit-a-monitor"
+  tag: "Documentation"
+  text: "Learn more about updating these settings via API"
+- link: "/dashboards/functions/interpolation/#default-zero"
+  tag: "Documentation"
+  text: "Learn more about interpolation"
+---
+
+## Overview
+
+When monitoring a metric that reports data infrequently, there are some tools and behaviors that should be considered to ensure a monitor's settings are appropriate for evaluating these queries as intended. 
+
+## Source of the monitor
+
+First, consider your source.
+
+### Metric-based monitor
+
+Is this a metric, change, anomaly or other metric-based monitor ? The following settings should be adjusted:
```

Comment:
What is "other metric-based monitor"? Can we list them out or keep them grouped under metric monitor?
You can go broad and reference out: "Is this a [metric-based monitor][reference link]?"
Or you can be very specific: "Metric-based monitors include X, Y, and Z. If your monitor is a metric based monitor, adjust the following settings:"

Reply:
*other data science based (outliers and forecast)